### PR TITLE
Add missing unpretty option help message

### PR DESCRIPTION
**Pull request by h-michael**
There are some missing help messages that is printed `ructc -Zunpretty help` and receiving invalid option.  

related with #16419, #45721, #21085, #31916

### CODE REVIEW

File: src/librustc/session/config.rs [hunk 1]
```diff
@@ -1342,10 +1342,16 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     unpretty: Option<String> = (None, parse_unpretty, [UNTRACKED],
         "Present the input source, unstable (and less-pretty) variants;
         valid types are any of the types for `--pretty`, as well as:
+        `expanded`, `expanded,identified`,
+        `expanded,hygiene` (with internal representations),
         `flowgraph=<nodeid>` (graphviz formatted flowgraph for node),
+        `flowgraph,unlabelled]` (unlabelled graphviz formatted flowgraph for node),
```
Comment (oli-obk) [inline src/librustc/session/config.rs]:
```suggestion
        `flowgraph,unlabelled=<nodeid>` (unlabelled graphviz formatted flowgraph for node),
```

File: src/librustc/session/config.rs [hunk 2]
```diff
@@ -1342,10 +1342,16 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
...
         `everybody_loops` (all function bodies replaced with `loop {}`),
-        `hir` (the HIR), `hir,identified`, or
-        `hir,typed` (HIR with types for each node)."),
+        `hir` (the HIR), `hir,identified`,
+        `hir,typed` (HIR with types for each node),
+        `hir-tree` (dump the raw HIR),
+        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)"
+        ),
```
Comment (oli-obk) [inline src/librustc/session/config.rs]:
```suggestion
    ),
```

File: src/librustc/session/config.rs [hunk 3]
```diff
@@ -1342,10 +1342,16 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
...
+        `expanded,hygiene` (with internal representations),
         `flowgraph=<nodeid>` (graphviz formatted flowgraph for node),
+        `flowgraph,unlabelled]` (unlabelled graphviz formatted flowgraph for node),
         `everybody_loops` (all function bodies replaced with `loop {}`),
-        `hir` (the HIR), `hir,identified`, or
-        `hir,typed` (HIR with types for each node)."),
+        `hir` (the HIR), `hir,identified`,
+        `hir,typed` (HIR with types for each node),
+        `hir-tree` (dump the raw HIR),
+        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)"
```
Comment (oli-obk) [inline src/librustc/session/config.rs]:
```suggestion
        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)",
```

**Review by oli-obk — CHANGES_REQUESTED** [review]
Just a few nits, lgtm otherwise

**Comment by h-michael** [comment]
@oli-obk Thanks for your so quick review.
I've fixed it.

**Review by oli-obk — APPROVED** [review]

File: src/librustc/session/config.rs (hunk 1 above)
Comment (oli-obk) [reply src/librustc/session/config.rs]: this has not been addressed, you accidentally edited the line above

File: src/librustc/session/config.rs [hunk 4]
```diff
@@ -1342,10 +1342,15 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
...
+        `expanded,hygiene` (with internal representations),
+        `flowgraph,unlabelled=<nodeid>` (unlabelled graphviz formatted flowgraph for node),
+        `flowgraph,unlabelled]` (unlabelled graphviz formatted flowgraph for node),
         `everybody_loops` (all function bodies replaced with `loop {}`),
-        `hir` (the HIR), `hir,identified`, or
-        `hir,typed` (HIR with types for each node)."),
+        `hir` (the HIR), `hir,identified`,
+        `hir,typed` (HIR with types for each node),
+        `hir-tree` (dump the raw HIR),
+        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)"),
```
Comment (oli-obk) [inline src/librustc/session/config.rs]:
```suggestion
        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)"
    ),
```

sorry about the earlier suggestion, I didn't know about the restriction of the macro

**Comment by h-michael** [comment]
@oli-obk Thanks for your review.
This is my first PR for rust.
So I don't  know so match review process.
According to CONTRIBUTING.md, do I had you write `@ bors +r`?
Or is there anything which I should do?

**Comment by oli-obk** [comment]
> According to CONTRIBUTING.md, do I had you write @ bors +r?

Once the PR is in a state where it can be merged, I will send it off to bors, yes. You don't need to worry about bors for now, I'll take care of that.

> Or is there anything which I should do?

I left a few comments, you can click the `Commit suggestion` button to apply the changes (or you can manually change the code on your computer if you prefer).

File: src/librustc/session/config.rs [hunk 5]
```diff
@@ -1342,10 +1342,15 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     unpretty: Option<String> = (None, parse_unpretty, [UNTRACKED],
         "Present the input source, unstable (and less-pretty) variants;
         valid types are any of the types for `--pretty`, as well as:
-        `flowgraph=<nodeid>` (graphviz formatted flowgraph for node),
+        `expanded`, `expanded,identified`,
+        `expanded,hygiene` (with internal representations),
+        `flowgraph,unlabelled=<nodeid>` (unlabelled graphviz formatted flowgraph for node),
```
Comment (oli-obk) [inline src/librustc/session/config.rs]:
```suggestion
        `flowgraph=<nodeid>` (unlabelled graphviz formatted flowgraph for node),
```

**Comment by h-michael** [comment]
> I left a few comments, you can click the Commit suggestion button to apply the changes (or you can manually change the code on your computer if you prefer).

Oh, I'm sorry, I overlooked that one. 🙇

**Comment by h-michael** [comment]
@oli-obk sorry, I fixed it.

**Comment by oli-obk** [comment]
@bors r+

Thanks!

**Pull request merged**